Libraries/DeprecatedPropTypes/DeprecatedTVViewPropTypes.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule DeprecatedTVViewPropTypes
+ * @flow
```

Comment:
Can you add @format as well so that prettier will format this file?

Reply:
should the `@providesModule` pragma be removed?

Reply:
Yeah, but it is fine to leave it in this diff. For some reason a bunch of the files in the repo still have this so we will remove them all in another PR. Better to be consistent for now. 👍